upgrade/breaking-changes/7-0-0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 7.0.0
+description: Handle the changes in the 7.0.0 release of the Telerik UI for Blazor components.
+page_title: Breaking Changes in 7.0.0
+slug: changes-in-7-0-0
+position: 970
+---
+
+# Breaking Changes in 7.0.0
+
+This article outlines the breaking changes in Telerik UI for Blazor version 7.0.0.
+
+## Common
+
+### .NET Support
+
+[.NET 7 is no longer supported]({%slug system-requirements%}).
+
+### AutoComplete, ComboBox, MultiColumnComboBox, MultiSelect
+
+The obsolete `ClearButton` parameter is removed. Use `ShowClearButton` instead.
+
+## DateRangePicker
+
+* The [`ShowOtherMonthDays` parameter's default value]({%slug daterangepicker-overview%}#daterangepicker-parameters) is changed to `false`.
+
+## DropDownButton
+
+* The [`ShowArrowButton` parameter's default value]({%slug dropdownbutton-overview%}#dropdownbutton-parameters) is changed to `false`.
+
+## FileSelect
+
+* The obsolete method `OpenFileSelectAsync()` is removed. [Use `OpenSelectFilesDialog()`]({%slug fileselect-overview%}#fileselect-reference-and-methods) instead.
+
+## Gantt
+
+* The [`Type` property of `GanttDependencyCreateEventArgs`]({%slug gantt-dependencies-editing%}) changes from `int` to [`GanttDependencyType` enum](/blazor-ui/api/telerik.blazor.ganttdependencytype).
+* The [dependency model class of the Gantt]({%slug gantt-dependencies-databind%}) must use the `GanttDependencyType` enum for its `TypeField` property.
+
+## Grid
+
+* Columns with `Visible="false"` are editable by default in popup edit mode. To avoid this, [set `Editable="false"` together with `Visible="false"`]({%slug components/grid/editing/popup%}).
```

Comment:
```suggestion
* Columns with `Visible="false"` are editable by default in popup edit mode. To avoid this, [set `Editable="false"` together with `Visible="false"`]({%slug components/grid/editing/popup%}).
```

This is not exactly correct. Currently the we render editors for all visible columns. Now we will render editor **only for Editable="true"** columns. The visibility of the column will not affect the editability.

Reply:
This statement targets developers that already have hidden columns and don't want them to be editable. I will reword.